From cca7fe56b7fced3c334d54c68a7d0890d695d567 Mon Sep 17 00:00:00 2001
From: Max Merchel <Max.Merchel@tq-group.com>
Date: Mon, 21 Jan 2019 09:05:44 +0100
Subject: [PATCH 068/188] serial: fsl_lpuart: add missing header consumer.h

fix [-Werror=implicit-function-declaration]
of function 'pinctrl_pm_select_sleep_state' and
of function 'pinctrl_pm_select_default_state'

Signed-off-by: Max Merchel <Max.Merchel@tq-group.com>
---
 drivers/tty/serial/fsl_lpuart.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/tty/serial/fsl_lpuart.c b/drivers/tty/serial/fsl_lpuart.c
index 421b0eb5f3e6..0755be17f101 100644
--- a/drivers/tty/serial/fsl_lpuart.c
+++ b/drivers/tty/serial/fsl_lpuart.c
@@ -26,6 +26,7 @@
 #include <linux/of.h>
 #include <linux/of_device.h>
 #include <linux/of_dma.h>
+#include <linux/pinctrl/consumer.h>
 #include <linux/pm_runtime.h>
 #include <linux/reset.h>
 #include <linux/serial_core.h>
